eng/scripts/Language-Settings.ps1

```diff
@@ -610,4 +612,9 @@ function Get-java-DocsMsMetadataForPackage($PackageInfo) {
     PreviewReadMeLocation = 'docs-ref-services/preview'
     Suffix = ''
   }
-}
\ No newline at end of file
+}
+
+function Validate-java-DocMsPackages ($PackageInfo, $DocValidationImageId) 
```

Comment:
`$DocValidationImageId` appears unused in both function signatures.

Reply:
It is a placeholder. Our java docker image is broken. I will fix the image first before we adding the real logic